packages/api-logs/index.jsx

```diff
@@ -222,8 +222,36 @@ class Logs extends React.Component {
   }
 
   render() {
-    const { query, baseUrl, group } = this.props;
-    if (!group) return null;
+    const { query, baseUrl, group, loginUrl } = this.props;
+    if (!group)
+      return (
+        <div className="logs">
+          <div>
```

Comment:
Should this have the `log-header` class?

Reply:
Nope! That class only [adds styles](https://github.com/readmeio/api-explorer/blob/2cd469a052a63fbc2c67386c3a49214e2573ea58/packages/api-logs/main.css#L25-L30) that make sense if there are logs present; e.g. a bottom border, spacing for this group selector dropdown (see below), etc.
![image](https://user-images.githubusercontent.com/8854718/81237191-9686cc80-8fc4-11ea-8b59-3c8eaa7a258c.png)